src/containers/ViewerPanel/index.tsx

```diff
@@ -485,7 +574,14 @@ class ViewerPanel extends React.Component<ViewerPanelProps, ViewerPanelState> {
                     onFollowObjectChanged={this.onSelectedAgentChange}
                 />
                 {firstFrameTime !== lastFrameTime && (
-                    <div className={styles.bottomControlsContainer}>
+                    <div
+                        className={classNames(
+                            styles.bottomControlsContainer,
+                            this.embedDisplaySettings.onlyRenderBottomControls
+                                ? styles.fullWidthBottomControls
+                                : styles.defaultControlSpacing
+                        )}
```

Comment:
These names are a bit confusing because minimized controls has been set up to be the deviation from the normal, but here you have both 'fullWidth' and "default". I think just checking the width here makes more sense than using the embedDisplaySettings.